omod/src/main/webapp/resources/htmlforms/Walkin-Client-Data-Capture-tool.xml

```diff
@@ -0,0 +1,256 @@
+                        }
+                    });                 
+
+                    jq("#reason-for-visit").change(function () {
+                        if (jq(this).find("select").find(":selected").val() == "90002") {
+                            enable_fields("reason-for-visit-other");
+                        } else {
+                            disable_fields("reason-for-visit-other");
+                        }
+                    });
+    
+                    beforeSubmit.push(function () {
+                        var submitVal = true;
+
+                        var submitValStage = [];
+                        var submitValStageART = [];
+
+                        if(getValue('telephone-number.value')==""){
+                            jq().toastmessage('showErrorToast', "Telephone Number is required");
+                        }
+            
+                        if(jq("#patientNumber").find("input:first").val()==""){
+                            jq().toastmessage('showErrorToast', "Patient Clinic number missing");
+                        }
+
+                        if(getValue('reason-for-visit.value')==""){
+                            jq().toastmessage('showErrorToast', "Reason for visit is required");
+                        }
+            
+                        if(getValue('current-regimen.value')==""){
+                            jq().toastmessage('showErrorToast', "Current Regimen is required");
+                        }
+
+                        if(getValue('parent-health-facility.value')==""){
+                            jq().toastmessage('showErrorToast', "Parent Health Facility is required");
+                        }
+
+                        if(getValue('serial-number.value')==""){
+                            jq().toastmessage('showErrorToast', "Serial Number is required");
+                        }
+
+                        if(getValue('pt-clinic-number.value')==""){
+                            jq().toastmessage('showErrorToast', "PT Clinic Number is required");
+                        }
+
+                        if(getValue('tb-status.value')==""){
+                            jq().toastmessage('showErrorToast', "The TB Status is required");
+                        }
+
+                        if(getValue('no-of-art-pills.value')==""){
+                            jq().toastmessage('showErrorToast', "The Art Refill is required");
+                        }
+
+                        if (submitValStage.indexOf(false) != -1) {
+                            submitVal = false;
+                            jq().toastmessage('showErrorToast', "There are errors during the validation of the data on the form. Please correct them and submit again");
+                        }
+
+                        return submitVal;
+                    });
+                    
+                   
+
+                });
+    
+            }
+    
+        </script>
+
+    <div>
```

Comment:
@ODORA0 Remove the outer border which is not needed as there are no tabs

Reply:
@ssmusoke Done

Reply:
@ODORA0 This has not been done see image below 
<img width="1754" alt="Screen Shot 2021-08-05 at 15 24 03" src="https://user-images.githubusercontent.com/689900/128349569-707c9622-6cda-463a-a440-f1689ba162c2.png">

Reply:
@ssmusoke Done